src/include/mpir_request.h

```diff
@@ -474,10 +474,10 @@ static inline void MPIR_Request_free_with_safety(MPIR_Request * req, int need_sa
 
         if (need_safety) {
             MPID_THREAD_CS_ENTER(POBJ, MPIR_THREAD_POBJ_HANDLE_MUTEX);
-            MPIR_Handle_obj_free_unsafe(&MPIR_Request_mem[pool], req);
+            MPIR_Handle_obj_free_unsafe(&MPIR_Request_mem[pool], req, FALSE);
```

Comment:
Add a comment `/* non-info objects */`, probably the line below as well

Reply:
Added `/* not info */` (`/* non-info objects */` makes these lines longer than 100 characters). Basically I added `/* not info */` or `/* info object */` to all `MPIR_Handle_obj_free_unsafe()` call sites.